apps/showcase/src/main/webapp/WEB-INF/decorators/main.jsp

```diff
@@ -61,7 +59,7 @@
     <meta name="description" content="Struts2 Showcase for Apache Struts Project">
     <meta name="author" content="The Apache Software Foundation">
 
-    <title><decorator:title default="Struts2 Showcase"/></title>
+    <title><sitemesh:write property="title"/></title>
```

Comment:
@lukaszlenart Don't we need a taglib replacement on L:52?

IntelliJ warns `Namespace 'sitemesh' is not bound`.

Reply:
Should be ok based on the doc
https://github.com/sitemesh/sitemesh3?tab=readme-ov-file#creating-a-decorator

Reply:
Thank you for the clarification!